src/temp/ProjectListControls/ProjectListControls.module.scss

```diff
@@ -0,0 +1,198 @@
+@import '../../theme/theme';
+
+.projectListControls {
+  background-color: $light;
+  height: 47px;
+  border-radius: 12px;
+  display: flex;
+  align-items: center;
+  max-width: 338px;
+  justify-content: space-between;
+  min-width: 288px;
+  box-shadow: 0px 4px 24px 0px rgba(var(--dark), 0.2);
```

Comment:
Could you replace the `var(--nameOfVariable)` syntax with `$nameOfVariable` syntax?